wcfsetup/install/files/lib/system/user/multifactor/TotpMultifactorMethod.class.php

```diff
@@ -78,6 +78,7 @@ public function createManagementForm(IFormDocument $form, ?Setup $setup, $return
                     ->label('wcf.user.security.multifactor.totp.code')
                     ->description('wcf.user.security.multifactor.totp.code.description')
                     ->required()
+                    ->autocomplete('one-time-code')
```

Comment:
This should be applied within `CodeFormField`.

Reply:
Are you sure? I'm asking because it is set to `off` explicitly in the constructor.

Reply:
Isn't terribly useful to set it to `off`, just to override it in all locations where the field is being used, no?

However: I should be verified whether `one-time-code` should be set in authentication only or in both authentication and registration. The MDN definition of this value is no terribly clear in how it's meant to be used and what the User Agent should do when seeing `one-time-code`.

Reply:
That's correct; but I don't have access to the other projects and didn't know why it was explicitly set to `off` before and where it might be used differently.

Reply:
The field is meant to be used for the use within the TOTP method only. That's why it sits within the TOTP namespace. If anyone uses it for its unrelated purpose then they are deep within warranty void territory.

The `off` value was selected, because the web browser's autocompletion is not useful here. The `one-time-code` value is so underspecified that it was not clear whether it helps, whether it hurts or whether it even does anything at all for TOTP (vs. e.g. SMS codes).